### ISSUE TITLE
Front page carousel

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#648]
new site

@djwesto just wondering if the headline text size had crept up in size at all? Seems to dominate a little more than it used to.

![image](https://cloud.githubusercontent.com/assets/6060985/7198949/76476e0c-e4e8-11e4-81f1-474b2162d125.png)

**Comment by pmackay** [comment]
Maybe you need to not use CAPS? :)

**Comment by GilesGibson** [comment]
BLEEDIN USERS!

**Comment by djwesto** [comment]
### I THINK @pmackay HAS PROBABLY HIT THE NAIL ON THE HEAD!!! I SUGGEST THERE'S NOT REALLY ANY NEED FOR THIS HEADLINE TO BE WRITTEN ALL IN CAPS, ALTHOUGH I PERSONALLY DO REALLY LOVE SYNCHRONISED SWIMMING.

There _are_ programmatic ways we could put in place to convert all-caps to mixed-case, but they'd strip out all caps except the first in a sentence, and, because it's often legitimate to use those for proper nouns etc, it would be much better to handle this sort of issue by subtly editing the users' original text to de-emphasise it.

**Comment by pmackay** [comment]
If we start coding fixes to decapitalize titles that users have typed in
all caps, something has gone really really wrong.....

**Comment by djwesto** [comment]
I agree. And I wouldn't do it, even if @GilesGibson asked for it because it would cause more problems than it solved. This is a clear example of where the editorial process simply needs to be strong enough to catch things like this.

**Comment by GilesGibson** [comment]
I would not be keen on having software re-write the headlines and decapitalise. Just thinking visually that the text can dominate, capitals or not. 

![image](https://cloud.githubusercontent.com/assets/6060985/7200474/4313ae08-e4f5-11e4-8c80-161b8e92c208.png)

I quite like the headline text and the teaser text all in the carousel. Maybe something for V1.1?

**Comment by djwesto** [comment]
I'm wondering what the next word is after _because of the the bloody [...]_

The trouble we're having with "teasers" is that we don't currently require them. Right now, we're not even asking authors to compose a teaser – we just pull in the first few words of their intro para and we need to improve how that works as well. A teaser would involve adding in an extra field to the node creation form which @pmackay and I were discussing a couple of days ago.

So feel free to create another **feature-request** issue in V1.1 if you like, but I would strongly suggest this isn't a 1.1 priority feature compared with some of the others.

**Comment by GilesGibson** [comment]
Happy to park this as a like to have for 1.1. I think the teaser text is but the first line(s) of the story and yes, sometimes it looks a little strange.
